core/src/avm2/globals/flash/ui/contextmenuitem.rs

```diff
@@ -0,0 +1,48 @@
+//! `flash.ui.ContextMenuItem` builtin/prototype
```

Comment:
In `avm1`, this file is called `context_menu_item.rs`, while in AVM2, there aren't many `_`s in file names... Not sure which variant would be more consistent - maybe it should be kept as is, even though this variation is a bit unfortunate.

Reply:
Yeah, by now, most avm2 globals' files have naming convention without `_`. I didn't contest this and just stayed with it so far.
(also, as extra argument, `i_bitmap_drawable.rs` looks kinda weird :) )

Reply:
Okay, that's fair... :D